src/main/java/HelloServlet.java

```diff
@@ -7,7 +9,9 @@
  */
 @WebServlet("/hello")
 public class HelloServlet extends HttpServlet {
+    private static final Logger LOGGER =Logger.getLogger(HelloServlet.class.getName());
     public void doGet (HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) throws IOException {
         httpServletResponse.getWriter().print("Hello, world!!!");
+        LOGGER.info("Request was gotten");
```

Comment:
there is no such word 'gotten'. It's better to log more meaningful information: so here you can specify which request was processed